WindowsServerDocs/identity/securing-privileged-access/securing-privileged-access.md

```diff
@@ -94,7 +94,7 @@ As an initial security measure for those users with Azure Active Directory and t
 
 ### 4. Identity attack detection
 
-[Azure Advanced Threat Protection (ATP)](/azure-advanced-threat-protection/what-is-atp) is a cloud-based security solution that identifies, detects, and helps you investigate advanced threats, compromised identities, and malicious insider actions directed at your on-premises Active Directory environment.
+[Microsoft Defender for Identity](https://docs.microsoft.com/en-us/defender-for-identity/what-is) is a cloud-based security solution that identifies, detects, and helps you investigate advanced threats, compromised identities, and malicious insider actions directed at your on-premises Active Directory environment.
```

Comment:
Fair enough. Just make sure to check whether we need to keep the /en-us/ part of the URL, because most of the documentation supports language agnostic page requests. In this case, it supports language agnostic page retrieval, but informs the reader if the language defined by their browser-specific location is not available by saying "This content is not available in your language. Here is the English version." I am fairly certain that most Docs pages can be translated to at least Spanish, French, and German (likely other languages too).
```suggestion
[Microsoft Defender for Identity](https://docs.microsoft.com/defender-for-identity/what-is) is a cloud-based security solution that identifies, detects, and helps you investigate advanced threats, compromised identities, and malicious insider actions directed at your on-premises Active Directory environment.
```